src/tests/main.cpp

```diff
@@ -98,7 +98,7 @@ int main(int argc, char* argv[])
 
       Botan_Tests::Test_Runner tests(std::cout);
 
-      return tests.run(opts);
+      return tests.run(opts) ? 0 : 1;
```

Comment:
Previously that returned the number of failed tests as the program's exit code. Keeping that behaviour would mean extra logic in the new reporter. I hope that doesn't matter too much.

Reply:
No it was just easy to do but I don't think anything cares except about pass/fail.